ramls/examples/circulation-rule-matches.json

```diff
@@ -1,15 +1,15 @@
 {
-  "loanRuleMatches": [
+  "circulationRuleMatches": [
```

Comment:
similarly, this could be `rulesMatches`? (and the below `ruleLine` etc)

Reply:
Changed in https://github.com/folio-org/mod-circulation/commit/4b0a55b49ecc627393e69877032f53a9554e7d7f